buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -363,16 +363,18 @@ class BeamModulePlugin implements Plugin<Project> {
     def aws_java_sdk_version = "1.11.519"
     def aws_java_sdk2_version = "2.5.71"
     def cassandra_driver_version = "3.6.0"
-    def generated_grpc_beta_version = "0.44.0"
-    def generated_grpc_ga_version = "1.43.0"
+    def generated_grpc_beta_version = "0.76.0"
+    def generated_grpc_ga_version = "1.75.0"
     def generated_grpc_dc_beta_version = "0.27.0-alpha"
     def google_auth_version = "0.12.0"
-    def google_clients_version = "1.28.0"
+    def google_http_clients_version = "1.32.0"
+    def google_oauth_clients_version = "1.30.2"
+    def google_api_client_version = "1.30.3"
     def google_cloud_bigdataoss_version = "1.9.16"
-    def google_cloud_core_version = "1.61.0"
-    def google_cloud_spanner_version = "1.6.0"
-    def grpc_version = "1.17.1"
-    def guava_version = "20.0"
+    def google_cloud_core_version = "1.91.1"
+    def google_cloud_spanner_version = "1.38.0"
+    def grpc_version = "1.23.0"
+    def guava_version = "28.1-jre"
```

Comment:
`Bigtable-client-core:1.12.1` needs Guava > 23.1 for `ImmutableList.builderWithExpectedSize`. 
Guava has 2 variation "-jre" for Java8 (stream/lambda support) and "-android" for Java7. `Gcsio:1.9.16` needs `ImmutableSet.toImmutableSet()` which is available in Guava "-jre".
https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1028#issuecomment-557633483